### DISCUSSION TITLE
I'd like to know what I can and can't do with self-hosting.

### DISCUSSION THREAD

**Discussion by sunecosuri** [renovatebot/renovate#9323] [Request Help]
Hi, I have found Renovate to be very useful. thanks!
I have one question.

I'd like to know what I can and can't do with self-hosting, Is there a comparison chart somewhere?

Is all that's written on https://docs.renovatebot.com/self-hosted-configuration/ ?

**Comment by rarkins** [comment]
When you are self-hosting, you have the power to change all of those self-hosted configuration options. When you are not, then you can't set any of them.

**Reply by sunecosuri** [reply]
@rarkins 
Thanks for the quick answer!
I'll change my question.
Is there any difference in functionality between SaaS (ex: Github Apps) and Self Hosting?
If there is a difference, is there a summary of the information somewhere?

**Reply by rarkins** [reply]
The core functionality, which you see in this repo, is the same. There is no further summary anywhere.

**Reply by sunecosuri** [reply]
Thank you, I understand!
